.github/workflows/benchmark.yml

```
@@ -0,0 +1,36 @@
+        with:
+          cli: latest
+
+      - name: Make benchmark script executable
+        run: chmod +x scripts/run_benchmark.sh
+
+      - name: Run SQL Protocol Benchmark
+        id: run-benchmark
+        run: |
+          mkdir -p benchmark-results
+          ./scripts/run_benchmark.sh
+        continue-on-error: true # Continue the workflow even if the benchmark fails
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)